examples/plot_credit_loan_decisions.py

```diff
@@ -66,17 +65,17 @@
 # same concept as anti-discrimination law. An AI system can comply with
 # anti-discrimination law while exhibiting fairness-related concerns. On the
 # other hand, some fairness interventions may be illegal under
-# anti-discrimination law. `Xiang and Raji (2019)
-# <https://arxiv.org/abs/1912.00761>`_ discuss the compatibilities and
+# anti-discrimination law. Xiang and Raji (2019)
```

Comment:
This is where you want to use the different `:footcite:` modes, to automatically generate the authors and year.